packages/app-store/closecom/pages/setup/index.tsx

```diff
@@ -140,7 +140,7 @@ export default function CloseComSetup() {
                       testPassed !== undefined ? (testPassed ? "test_passed" : "test_failed") : "test_api_key"
                     )}
                   </Button>
-                  <Button type="submit" loading={form.formState.isSubmitting}>
+                  <Button color="secondary" type="submit" loading={form.formState.isSubmitting}>
```

Comment:
The save button should stay primary here, but it looks like dark mode is not working correctly. Same happens for the zapier setup page: 
<img width="762" alt="Screenshot 2023-05-08 at 13 53 42" src="https://user-images.githubusercontent.com/30310907/236817251-2295da13-cb0a-422c-babb-c4b3d047eab1.png">

@sean-brydon Any idea why the dark mode for the primary button is not working here?

Reply:
There is a PR up to fix that: https://github.com/calcom/cal.com/pull/8758